src/sentry/pipeline/views/base.py

```diff
@@ -1,34 +1,35 @@
+from __future__ import annotations
+
 import abc
 from collections.abc import Mapping
 from typing import TYPE_CHECKING, Any
 
+from django.http.request import HttpRequest
 from django.http.response import HttpResponseBase
-from rest_framework.request import Request
 
 from sentry.utils import json
-from sentry.web.frontend.base import BaseView
 from sentry.web.helpers import render_to_response
 
 if TYPE_CHECKING:
     from sentry.pipeline.base import Pipeline
 
 
-class PipelineView(BaseView, abc.ABC):
+class PipelineView(abc.ABC):
```

Comment:
this is the actual change -- the rest is just matching the function signature of `def dispatch`